src/util-affinity.c

```diff
@@ -111,12 +111,12 @@ void BuildCpusetWithCallback(const char *name, ConfNode *node,
         } else if (strchr(lnode->val, '-') != NULL) {
             char *sep = strchr(lnode->val, '-');
             char *end;
-            a = strtoul(lnode->val, &end, 10);
+            a = (int)strtoul(lnode->val, &end, 10);
```

Comment:
this is a bit odd, downcasting `strtoul` to int... should we just use atoi? Or one of our parsing wrappers perhaps

Reply:
Trying StringParseUint32